### PR TITLE
Update saving-data.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Working my way through the ORM related documenation trying to make some (hopefully) readability improvements. Please let me know if I should tweak anything. Thanks!

### CODE REVIEW

File: en/orm/saving-data.rst [hunk 1]
```diff
@@ -31,25 +31,25 @@ passing it to the ``save()`` method in the ``Table`` class::
...
 
-Updating is equally easy, and the ``save()`` method is also used for that
+Updating your data is equally easy, and the ``save()`` method is also used for that
 purpose::
 
     use Cake\ORM\TableRegistry;
 
     $articlesTable = TableRegistry::get('Articles');
-    $article = $articlesTable->get(12); // article with id 12
+    $article = $articlesTable->get(12); // Return article with id 12
```
Comment (ravage84) [inline en/orm/saving-data.rst]: [this/it] Returns

File: en/orm/saving-data.rst [hunk 2]
```diff
@@ -136,36 +136,37 @@ that they can be validated and saved.
...
-            ['body' => 'Second comment'],
+            ['body' => 'The CakePHP features are outstanding'],
+            ['body' => 'CakePHP performance is terrific!'],
         ]
     ];
 
 By default, the ``newEntity()`` method validates the data that gets passed to
 it, as explained in the :ref:`validating-request-data` section. If you wish to
-prevent data from being validated, pass the ``'validate' => false`` option::
+bypass data validation pass the ``'validate' => false`` option::
```
Comment (ravage84) [inline en/orm/saving-data.rst]: "bypass the data validation, set the"
+the
-pass
+set the

File: en/orm/saving-data.rst [hunk 3]
```diff
@@ -136,36 +136,37 @@ that they can be validated and saved.
...
     $articles = TableRegistry::get('Articles');
+    
     // Validate and convert to an Entity object
     $entity = $articles->newEntity($this->request->data());
 
 The request data should follow the structure of your entities. For example if
-you had an article, which belonged to a user, and had many comments, your
-request data should look like::
+you have an article, which belonged to a user, and had many comments, your
+request data should resemble::
```
Comment (ravage84) [inline en/orm/saving-data.rst]: Even though "resemble" is better English, I think "look like" was OK.

**Comment by ravage84** [comment]
Thanks, man! Keep up the good work.

File: en/orm/saving-data.rst (hunk 2 above)
Comment (markstory) [inline en/orm/saving-data.rst]: I think the current wording is fine.

**Comment by jeffblack360** [comment]
Thanks for the comments guys!  I've just started working my way through all the ORM documentation so keep your eyes out for more PRs, and don't hesitate to point me in the right direction when I go overboard with anything.

**Pull request merged**